src/rkr/ui/rkr.cc

```diff
@@ -150,12 +150,16 @@ int main(int argc, char* argv[]) noexcept {
   build->add_option("-o,--output", command_output,
                     "Output file where commands should be printed (default: -)");
 
+  // Flags for rkr with remote connections
+  string remote_path = "-";
```

Comment:
The "-" default usually means standard output, so that doesn't make sense here. This may be a place to use `std::optional` so there is a possibility of no value.